.github/docker-compose.yaml

```
@@ -11,5 +11,7 @@ services:
         container_name: dask-worker
         image: daskdev/dask:latest
         command: dask-worker dask-scheduler:8786
+        environment:
+            EXTRA_CONDA_PACKAGES: "pyarrow>1.0.0"
```

Would probably add a comment here briefly summarizing the need - we'll probably consult this later when documenting cluster requirements:

```suggestion
            EXTRA_CONDA_PACKAGES: "pyarrow>1.0.0"  # required for parquet read/writes
```